generators/client/templates/angular/src/main/webapp/app/account/activate/_activate.component.ts

```diff
@@ -21,7 +21,7 @@ import { NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
 import { ActivatedRoute } from '@angular/router';
 
 import { ActivateService } from './activate.service';
-import { LoginModalService } from '../../shared';
+import { LoginModalService } from '../../core';
```

Comment:
please note that I merged folder restructure PR to master. Now you can do imports for this like `app/core` instead of providing relative paths. see if you can use that everywhere